core/pom.xml

```diff
@@ -241,6 +241,23 @@
             <version>2.10.5</version>
         </dependency>
 
+        <dependency>
```

Comment:
The same dep is appearing twice (probs because of the rebases). You can keep one and clean up the whitespaces.
Also, you might consider using a more up to date version of breze, like the 0.11